CSharp/Samples/AlarmBot/Models/AlarmModule.cs

```diff
@@ -20,7 +20,7 @@ protected override void Load(ContainerBuilder builder)
         {
             base.Load(builder);
 
-            builder.Register(c => new LuisModelAttribute("c413b2ef-382c-45bd-8ff0-f76d60e2a821", "6d0966209c6e4f6b835ce34492f3e6d9")).AsSelf().SingleInstance();
+            builder.Register(c => new LuisModelAttribute("c413b2ef-382c-45bd-8ff0-f76d60e2a821", "6d0966209c6e4f6b835ce34492f3e6d9")).AsSelf().AsImplementedInterfaces().SingleInstance();
```

Comment:
I don't think we need to add the constructor overload, but can you fix the other registration in the BotBuilder\CSharp\Samples\AlarmBot\Models\AlarmModule.cs sample?  i.e. add AsImplementedInterfaces

            builder.Register(c => new LuisModelAttribute("c413b2ef-382c-45bd-8ff0-f76d60e2a821", "6d0966209c6e4f6b835ce34492f3e6d9")).AsSelf().SingleInstance();